charts/optimize-live/templates/deployment.yaml

```diff
@@ -38,6 +34,10 @@ spec:
           envFrom:
           - secretRef:
               name: {{ include "optimize-live.fullname" . }}
+{{- if .Values.debug }}
+          args:
+            - --zap-log-level=9
```

Comment:
Just noticed that this should use Kube indentation conventions (i.e. the `-` should align with the `a` in `args`).

Reply:
good call, fixed.